config/jobs/kubernetes-sigs/cluster-api-provider-gcp/OWNERS

```diff
@@ -19,3 +18,6 @@ emeritus_approvers:
 - maisem
 - detiber
 - krousey
+- richardcase
+- justinsb
```

Comment:
I think @justinsb was planning to keep being a reviewer if I understood correctly